### PR TITLE
Include gluster-prometheus-exporter rpm in glusterd2 container

### PR DESCRIPTION
**Pull request by aravindavk**
- Depends on rpm spec file PR
  https://github.com/gluster/gluster-prometheus/pull/26
- Nightly rpm building is not yet setup
- Depends of environment variable related PR
  https://github.com/gluster/gluster-prometheus/pull/66

Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

**Comment by aravindavk** [comment]
retest this please

**Comment by aravindavk** [comment]
Nightly rpm is now setup http://artifacts.ci.centos.org/gluster/gluster-prometheus-nightly/ (centosci PR: https://github.com/gluster/centosci/pull/36)

Removed `DO_NOT_MERGE` label, Please review

File: extras/nightly-container/provision.yml [hunk 1]
```diff
@@ -111,3 +113,35 @@
       args:
         warn: no
 
+    - name: Configure gluster-exporter to use glusterd2
+      replace:
+        path: /etc/gluster-exporter/gluster-exporter.toml
+        regexp: 'gluster-mgmt = "glusterd"'
```
Comment (JohnStrunk) [inline extras/nightly-container/provision.yml]: May want to make these matches a bit more robust ala: `'^gluster-mgmt =.*'` (or something alone those lines)... Just to ensure the replacement happens even if the default options changes subtly.

Comment (aravindavk) [reply extras/nightly-container/provision.yml]: Fixed as suggested.

File: extras/nightly-container/provision.yml [hunk 2]
```diff
@@ -111,3 +113,35 @@
...
+      ini_file:
+        path: /etc/systemd/system/gluster-exporter.service.d/override.conf
+        section: Service
+        option: PassEnvironment
+        # PassEnvironment requires that each variable be mentioned individually
+        # More variables will need to be added here as required
+        value: GD2_ENDPOINTS
+
+    - name: Enable gluster-exporter.service
+      command: systemctl enable gluster-exporter.service
```
Comment (kshlm) [inline extras/nightly-container/provision.yml]: You should be using the service module instead of the command module. Any reason not to?

Comment (aravindavk) [reply extras/nightly-container/provision.yml]: referred other instance in the same file(Line 109)

**Comment by aravindavk** [comment]
Updated to use "service" module instead of "command". @kshlm please review

**Pull request merged**